content/en/admin/install.md

```diff
@@ -13,7 +13,7 @@ menu:
 * A **domain name** \(or a subdomain\) for the Mastodon server, e.g. `example.com`
 * An e-mail delivery service or other **SMTP server**
 
-You will be running the commands as root. If you aren’t already root, switch to root:
+You will be running the commands as root. If you aren’t already root, switch to root: `sudo su -`
```

Comment:
Is the hyphen always required? I think I've always done this with just `sudo su`.

Reply:
not sure, but it works for me